data/presets/fields/address.json

```diff
@@ -0,0 +1,64 @@
+{
+    "type": "address",
+    "keys": [
+        "addr:block_number",
+        "addr:city",
+        "addr:block_number",
+        "addr:conscriptionnumber",
+        "addr:county",
+        "addr:country",
+        "addr:county",
+        "addr:district",
+        "addr:floor",
+        "addr:hamlet",
+        "addr:housename",
+        "addr:housenumber",
+        "addr:neighbourhood",
+        "addr:place",
+        "addr:postcode",
+        "addr:province",
+        "addr:quarter",
+        "addr:state",
+        "addr:street",
+        "addr:subdistrict",
+        "addr:suburb"
+    ],
+    "reference": {"key": "addr"},
+    "icon": "address",
+    "universal": true,
+    "label": "Address",
+    "strings": {
```

Comment:
I _think_ there's an external set of translations that iD merges at build-time. (@gmaclennan's preset tools do something to manage translations, but I haven't looked deeply enough yet.)

This file is as-is from iD's presets, so I'd be inclined to do the transformation at build / runtime to ensure that the localization module gets what it needs.